src/introductory-offer-banner/index.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * External dependencies
+ */
+import { Fill, Card, Button } from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+import { OPTIONS_STORE_NAME } from '@woocommerce/data';
+import { useSelect, useDispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+export const WC_CALYPSO_BRIDGE_INTRODUCTORY_OFFER_BANNER_HIDDEN =
+	'wc_calypso_bridge_introductory_offer_banner_hidden';
+
+export const CalypsoBridgeIntroductoryOfferBanner = () => {
+	const { updateOptions } = useDispatch( OPTIONS_STORE_NAME );
+	const { isModalHidden } = useSelect( ( select ) => {
+		const { getOption, hasFinishedResolution } =
+			select( OPTIONS_STORE_NAME );
+
+		return {
+			isModalHidden:
+				getOption(
+					WC_CALYPSO_BRIDGE_INTRODUCTORY_OFFER_BANNER_HIDDEN
+				) === 'yes' ||
+				! hasFinishedResolution( 'getOption', [
+					WC_CALYPSO_BRIDGE_INTRODUCTORY_OFFER_BANNER_HIDDEN,
+				] ),
+		};
+	} );
+
+	const dismissModal = () => {
+		updateOptions( {
```

Comment:
I just thought we may want to fire a track for dismiss.

Lets add `recordEvent( 'free_trial_dismiss_homescreen_intro_offer_banner' )`. What do you think?

Reply:
Updated in https://github.com/Automattic/wc-calypso-bridge/pull/1397/commits/6ec02a5c7557857dd1aab523d31cb02bf3091f8f

Reply:
Thanks!